activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/resources/application.properties

```diff
@@ -20,3 +20,5 @@ keycloak.principal-attribute=preferred-username
 activiti.keycloak.admin-client-app=admin-cli
 activiti.keycloak.test-user=client
 activiti.keycloak.test-password=client
+
+activiti.cloud.query.uri-prefix=
```

Comment:
You should not need to have to define this in test properties if there is default properties source defined for each service

Reply:
we removed these properties